standard/docs/en/_static/extensions.js

```diff
@@ -20,17 +24,16 @@ $(function () {
 
   // Schema reference -- extensions 
   if (isSchemaReferencePage) {
-    var blankListEl = '<dl><dt><a class="reference external" href=""></a></dt><dd></dd></dl>';
     // append an empty list for community part of the section
     $('.extension_list p.last').after('<br><dl class="docutils hide community-list"></dl>');
     $('.extension_list p.last').wrap('<small></small>');
   }
 
+  // Get the community extensions to add to the documentation.
   $.ajax({
-    "url": "http://standard.open-contracting.org/extension_registry/master/extensions.js", 
-    "jsonpCallback": "extensions_callback",
+    "url": "https://raw.githubusercontent.com/open-contracting/extension_registry/master/build/extensions.json",
```

Comment:
We were trying to make sure we did not have a reliance on github.

Reply:
We also have to commit a compiled version of the extensions to the extension_registry which was also something that we tried to avoid.

Reply:
Indeed, this is a stopgap.